src/sentry/mail/actions.py

```diff
@@ -48,12 +49,16 @@ def after(self, event, state):
         if features.has(
             "organizations:issue-alert-fallback-targeting", self.project.organization, actor=None
         ):
-            fallthrough_type = FallthroughChoiceType(self.data["fallthroughType"])
+            fallthrough_choice = self.data.get("fallthroughType", None)
+            fallthrough_type = (
```

Comment:
I think you can do `fallthrough_type = (FallthroughChoiceType(fallthrough_choice) or None`